simpleflow/task.py

```diff
@@ -75,6 +82,27 @@ def __init__(self, activity, *args, **kwargs):
         self.kwargs = self.resolve_kwargs(**kwargs)
         self.id = None
 
+    def load_middlewares(self, middlewares):
+        if not middlewares:
+            return
+            
+        for pre in middlewares["pre"]:
+            try:
+                func = import_from_module(pre)
+            except AttributeError:
+                logger.exception("Cannot import a pre middleware from {}".format(pre))
```

Comment:
```suggestion
                logger.exception("Cannot import a pre middleware from %r", pre)
```
(letting logger.exception do the formatting)